airbrakes/state.py

```diff
@@ -173,8 +175,12 @@ def update(self):
 
         data = self.context.data_processor
 
-        # If our altitude is 0, we have landed:
-        if data.current_altitude <= GROUND_ALTITUDE:
+        # If our altitude and speed are around 0, we have landed
+        if data.current_altitude <= GROUND_ALTITUDE and abs(data.vertical_velocity) <= LANDED_SPEED:
+            self.next_state()
+
+        # If we have been in free fall for too long, we move to the landed state
+        if convert_to_seconds(data.current_timestamp - self.start_time_ns) >= MAX_FREE_FALL_LENGTH:
```

Comment:
can we have this in combination with altitude < 15 just to be extra safe

Reply:
that kinda defeats the point of the fallback IMO

Reply:
ok. I was imagining a scenario where the chutes are deployed and the gust is so strong that it takes a really long time to land

Reply:
I talked to trent, he said the max amount of time he could ever see it being in the air (even if winds were insane) would be 180 seconds, so I'm just gonna make it that.